CONTRIBUTORS.md

```diff
@@ -5,7 +5,22 @@
 Thanks for all your contributions :heart: :octocat:
 
 
-[prateekiiest](https://github.com/prateekiiest) : KWOC Mentor
+[prateekiiest](https://github.com/prateekiiest) : KWOC Mentor <br>
+[Rahul Arulkumaran](https://github.com/rahulkumaran) : KWOC Contributor <br>
 
 | Github username      | Pull Request           | Status  |
 | ------------- |:-------------:| -----:|
+|[Rahul Arulkumaran](https://github.com/rahulkumaran)| [#46](https://github.com/codeIIEST/Algorithms/pull/46) | CLOSED |
+|[Rahul Arulkumaran](https://github.com/rahulkumaran)| [#66](https://github.com/codeIIEST/Algorithms/pull/66) | CLOSED |
+|[Rahul Arulkumaran](https://github.com/rahulkumaran)| [#88](https://github.com/codeIIEST/Algorithms/pull/88) | CLOSED |
+|[Rahul Arulkumaran](https://github.com/rahulkumaran)| [#89](https://github.com/codeIIEST/Algorithms/pull/89) | CLOSED |
+|[Rahul Arulkumaran](https://github.com/rahulkumaran)| [#90](https://github.com/codeIIEST/Algorithms/pull/90) | CLOSED |
+|[Rahul Arulkumaran](https://github.com/rahulkumaran)| [#93](https://github.com/codeIIEST/Algorithms/pull/93) | CLOSED |
```

Comment:
instead of so many names, add the PR links separated by commas, do it only in one field like this

name  | PR1, PR2, PR3 | Open, Closed, Merged

Reply:
SO I'll create 3 rows? 1 each for open, closed and merged! Is that okay? @prateekiiest